config.yaml

```diff
@@ -45,6 +45,16 @@ listeners:
       databaseId: YOUR_SPANNER_DATABASE
       # [Optional] - Global else default to TableConfigurations
       configTableName: TableConfigurations
+      # [Optional] endpoint configuration for spanner
```

Comment:
This should be setting in `cassandra_to_spanner_configs` which is global settings for all listeners. Not sure if you want to provide per listeners override settings?